src/Chapter1.hs

```diff
@@ -636,7 +651,10 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n
+    | n > 0 && n < 10 = n
+    | otherwise = firstDigit (div n 10)
```

Comment:
The implementation is correct, but it works only for positive numbers. You probably can use one of the familiar functions to handle negative numbers :slightly_smiling_face: